tests/e2e/api/digitalAsset.test.ts

```diff
@@ -252,9 +255,10 @@ describe("DigitalAsset", () => {
         "U256 key",
         "ADDRESS key",
         "STRING key",
+        "ARRAY KEY",
       ],
-      propertyTypes: ["BOOLEAN", "U8", "U16", "U32", "U64", "U128", "U256", "ADDRESS", "STRING"],
-      propertyValues: [true, 1, 1, 1, 1, 1, 1, bob.accountAddress, "hi"],
+      propertyTypes: ["BOOLEAN", "U8", "U16", "U32", "U64", "U128", "U256", "ADDRESS", "STRING", "ARRAY"],
+      propertyValues: [true, 1, 1, 1, 1, 1, 1, bob.accountAddress, "hi", "[hello, nice, world]"],
```

Comment:
So, can you only provide vector<u8> as a string then?  Since byte arrays aren't allowed?

Reply:
this is what we allow in the remoteABI parser for `vector<u8>`. To support an array with different value types (i.e `"[hello, 1, true, world]"` we need to pass it as a string.
```
if (param.value.isU8()) {
      // We don't allow vector<u8>, but we convert strings to UTF8 uint8array
      // This is legacy behavior from the original SDK
      if (isString(arg)) {
        return MoveVector.U8(TEXT_ENCODER.encode(arg));
      }
      if (arg instanceof Uint8Array) {
        return MoveVector.U8(arg);
      }
      if (arg instanceof ArrayBuffer) {
        return MoveVector.U8(new Uint8Array(arg));
      }
    }
```

Reply:
It's a little bit stranger than that though, it's not really an array of values, but instead a string `[hello, nice, world]`.  It doesn't parse the brackets and all into an array.

This means I can't put arbitrary bytes, but only a string in `vector<u8>`